src/models/fields/IllPerson.schema.ts

```diff
@@ -0,0 +1,27 @@
+import Joi from '@hapi/joi';
+
+import { IllPerson } from 'app/models/fields/IllPerson';
+
+export const emptySelectionErrorMessage = 'You must select a person';
+export const emptyOtherPersonErrorMessage = 'You must tell us more information';
+
+export const schema = Joi.object({
+    illPerson: Joi.string()
+        .required()
+        .valid(...Object.values(IllPerson).filter(x => typeof x === 'string'))
+        .messages({
+            'any.required': emptySelectionErrorMessage,
+            'any.only': emptySelectionErrorMessage,
+            'string.base': emptySelectionErrorMessage,
+            'string.empty': emptySelectionErrorMessage
```

Comment:
I noticed you are checking the type here (any and string). Isn't this for radio buttons? Why are we checking string.base, string.empty

Reply:
People can edit radio buttons through the developer console. But... come to think of it, only a preset list of values are valid anyway, so these two lines are indeed redundant. Nice spot.

Reply:
make sure to double check it all works fine